SofaKernel/framework/sofa/core/visual/DrawToolGL.cpp

```diff
@@ -1027,8 +1063,9 @@ void DrawToolGL::drawEllipsoid(const Vector3 &p, const Vector3 &radii)
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-void DrawToolGL::drawBoundingBox( const Vector3 &min, const Vector3 &max )
+void DrawToolGL::drawBoundingBox( const Vector3 &min, const Vector3 &max, float size)
 {
+    glLineWidth(size);
```

Comment:
this will break code that have set the drawLine before.
cannot push/pop the attrib otherwise ?

Reply:
The normal convention is to save the attributes before changing them and restoring them to their old value when you are done. In your case you restore it to 1.0 regardless of its initial value this means that calling the function has a side effect on all future opengl drawing operation. It is way better to have side effect free functions so that the one that call them does not have to care about what the function is doing.